src/Gui/Placement.cpp

```diff
@@ -49,6 +49,11 @@
 #include "Placement.h"
 #include "ui_Placement.h"
 
+#include <Gui/MainWindow.h>
+#include <Gui/View3DInventor.h>
+#include <Inventor/SbVec3f.h>
+#include <Gui/SpaceballEvent.h>
+#include <Base/DualQuaternion.h>

```

Comment:
This header seems to be superfluous

Reply:
Yes, every thing except 
`#include <Gui/SpaceballEvent.h>`
seems superfluous and are left from old experiments.